src/bridge/AbsBridge.sol

```diff
@@ -203,6 +203,7 @@ abstract contract AbsBridge is Initializable, DelegateCallAware, IBridge {
             baseFeeL1,
             blockTimestamp
         );
+        emit MessageDeliveredV2(kind, sender);
```

Comment:
Might make more sense to have this event in the depositETH function so it does not add gas usage to other types of inbox messages. Can you elaborate more on the usecase of this feature?

Reply:
The main usecase is for a bridge application to show a user their deposits. Normally the MessageDelivered event is used for this, however since address is not indexed in this event, deposits for all users have to be queried which is much less efficient and requires more requests. Agreed that the event only needs to be emitted for deposit events - updated the logic to reflect that, which should save on gas.